.github/workflows/wheels.yml

```diff
@@ -239,10 +239,7 @@ jobs:
         uses: actions/setup-python@v4
 
       - name: Upload artifacts
-        env:
-          # Secret variables need to be mapped to environment variables explicitly
-          SCIKIT_LEARN_NIGHTLY_UPLOAD_TOKEN: ${{ secrets.SCIKIT_LEARN_NIGHTLY_UPLOAD_TOKEN }}
-          SCIKIT_LEARN_STAGING_UPLOAD_TOKEN: ${{ secrets.SCIKIT_LEARN_STAGING_UPLOAD_TOKEN }}
-          ARTIFACTS_PATH: dist/artifact
-        # Force a replacement if the remote file already exists
-        run: bash build_tools/github/upload_anaconda.sh
+        uses: scientific-python/upload-nightly-action@b67d7fcc0396e1128a474d1ab2b48aa94680f9fc # 0.5.0
```

Comment:
Is the use of a hash + version number comment an artifact of the use of dependabot to avoid having silent tag renaming not being reflected in our version control?

Reply:
It's not an artifact of dependabot usage.

- Here the hash, i.e., full length commit SHA signifies a secure way to use action provided by [scientific-python](https://github.com/scientific-python). ```#0.5.0``` is the current release of [upload-nightly-action](https://github.com/scientific-python/upload-nightly-action). This hash is manually updated for each release, for example by [this PR](https://github.com/scientific-python/upload-nightly-action/pull/69). More information about the full commit length commit SHA can be found [here](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions).
- Dependabot is used to automatically update non-default actions for a specified interval, here is weekly. More details about this tool can be found [here](https://learn.scientific-python.org/development/guides/gha-basic/#updating).

Reply:
Thanks for the clarification. Maybe we could enable dependabot for github actions in a dedicated PR. I have no experience with using dependabot for that purpose I would appreciate what other maintainers about this change.